src/algebra/ordered_group.lean

```diff
@@ -532,6 +532,23 @@ instance with_top.canonically_ordered_add_monoid : canonically_ordered_add_monoi
 
 end canonically_ordered_add_monoid
 
+/-- A canonically ordered monoid is an ordered commutative monoid
+  in which the ordering coincides with the divisibility relation,
+  which is to say, `a ≤ b` iff there exists `c` with `b = a + c`.
+  This is satisfied by the natural numbers, for example, but not
+  the integers or other ordered groups. -/
+@[protect_proj]
+class canonically_linear_ordered_add_monoid (α : Type*)
+      extends canonically_ordered_add_monoid α, decidable_linear_order α
```

Comment:
I'm really confused by the discrepancies between the doc string and the definition.

Oh -- you've copied and pasted the doc-string from `canonically_ordered_add_monoid`. Can you please fix this? While you're at it, can you insert the what "additive" at the two appropriate places in the doc-string for `canonically_ordered_add_monoid`?